### PR TITLE
libupnp: update to 1.14.14

### PR DESCRIPTION
**Pull request by PolynomialDivision**
Changes:
- http: RecvMessage: Fix potential allocation failure

Signed-off-by: Nick Hainke <vincent@systemli.org>

Maintainer: @thess

### CODE REVIEW

File: libs/libupnp/Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libupnp
-PKG_VERSION:=1.14.12
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_VERSION:=1.14.13
+PKG_RELEASE:=1
```
[the remark this answers is not in the document]
Comment (PolynomialDivision) [reply libs/libupnp/Makefile:5]: Whups, that was by accident. Package repository was old, so I stashed, and popped again, and had a merge conflict. I did not see that.

**Comment by PolynomialDivision** [comment]
Fails because of:
```
CMake Error at CMakeLists.txt:10 (include):
  include could not find requested file:

    cmake/test-functions.cmake


CMake Error at CMakeLists.txt:11 (include):
  include could not find requested file:

    cmake/autoheader.cmake


CMake Warning at CMakeLists.txt:13 (project):
  VERSION keyword not followed by a value or was followed by a value that
  expanded to nothing.


CMake Error at CMakeLists.txt:16 (include):
  include could not find requested file:

    cmake/options.cmake


CMake Error: File /home/build/openwrt/build_dir/target-arc_archs_glibc/libupnp-1.14.13/autoconfig.h.cm does not exist.
CMake Error at CMakeLists.txt:250 (configure_file):
  configure_file Problem configuring file


CMake Error at /home/build/openwrt/staging_dir/host/share/cmake-3.22/Modules/WriteBasicConfigVersionFile.cmake:43 (message):
  No VERSION specified for WRITE_BASIC_CONFIG_VERSION_FILE()
Call Stack (most recent call first):
  /home/build/openwrt/staging_dir/host/share/cmake-3.22/Modules/CMakePackageConfigHelpers.cmake:237 (write_basic_config_version_file)
  CMakeLists.txt:266 (write_basic_package_version_file)


-- Configuring incomplete, errors occurred!
```

**Comment by neheb** [comment]
Yes it does. Upstream is bad at maintaining their build systems.

**Comment by neheb** [comment]
14 is available.

**Comment by PolynomialDivision** [comment]
Did not change anything. I will close.

**Comment by neheb** [comment]
Alpine has this patch: https://github.com/alpinelinux/aports/blob/master/community/libupnp/disable-failing-test.patch

**Pull request closed without merging**